tests/samplers_tests/test_samplers.py

```diff
@@ -1035,3 +1042,6 @@ def test_reproducible_in_other_process(sampler_class_index: int) -> None:
     assert not (hash_dict[0] == hash_dict[1] == hash_dict[2])
     # But the sequences are expected to be the same.
     assert sequence_dict[0] == sequence_dict[1] == sequence_dict[2]
+
+    if hash_seed is not None:
+        os.environ["PYTHONHASHSEED"] = hash_seed
```

Comment:
Could you move these lines before asserts? If the assertion fails, `PYTHONHASHSEED` does not back to the original.

```suggestion
    if hash_seed is not None:
        os.environ["PYTHONHASHSEED"] = hash_seed

    assert not (hash_dict[0] == hash_dict[1] == hash_dict[2])
    # But the sequences are expected to be the same.
    assert sequence_dict[0] == sequence_dict[1] == sequence_dict[2]
```

Reply:
Thank you for your comment and suggestions!

Reply:
The comment (This test should be~) doesn't match the code. Sorry, it is my mistake.

Reply:
Thanks. I might misunderstand your comment, but I've slightly updated the comments not to specify `tox` case. What do you think?

Reply:
LGTM!